modules/nf-core/dupradar/templates/dupradar.r

```diff
@@ -124,19 +124,13 @@ line="#id: dupradar
 #        - color: 'green'
 #          dash: 'LongDash'
 #          label:
-#                style: {color: 'green'}
```

Comment:
Since this is an nf-core module we should really be fixing this in nf-core modules, or via the patch currently on dev (which I just noticed has some config bits I should probably remove)

Reply:
Fair enough. I did admittedly not consider whether that was a local file or is provided with a module. But to change it in the module, I would prefer seeing the results first, so wait for a MultiQC version that is working for custom content again.

Reply:
However, I noticed that @drpatelh also [changed that file locally for the pipeline](https://github.com/nf-core/rnaseq/commit/6035f5b81435f31fcdc4ff0758736b96f1eed54b). So I think we would redo that for the module once we tested everything with a new MultiQC version.

Reply:
Yep, made a patch for that one

Reply:
Ok, I have now undone all changes and reverted to your file version from when you turned it into a module.